modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -485,6 +486,7 @@ private ObjectField _addObjectField(
 		ObjectDefinition objectDefinition =
 			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 
+		_validateDefaultValue(businessType, defaultValue);
```

Comment:
I will move down this line to below the `_validateName(0, objectDefinition, name, system);`